src/prefect/cli/project.py

```diff
@@ -35,9 +37,29 @@ async def ls():
     recipe_paths = (
         Path(prefect.__root_path__) / "src" / "prefect" / "projects" / "recipes"
     )
-    recipes = [p.name for p in recipe_paths.iterdir() if p.is_dir()]
-    for recipe in recipes:
-        app.console.print(f"[green]{recipe}[/green]")
+    recipes = {}
+
+    for recipe in recipe_paths.iterdir():
+        if recipe.is_dir():
+            with open(recipe / "prefect.yaml") as f:
```

Comment:
Can you add handling for the case where there isn't a `prefect.yaml` file present to avoid crashing the command in case we forget one on accident?

Reply:
Yup can do

Reply:
@desertaxle updated